packages/evo/src/agent-functions/executeScript.ts

```diff
@@ -115,7 +115,12 @@ export const executeScript: AgentFunction<AgentContext> = {
         }
 
         let args: any = undefined;
-        args = params.arguments ? params.arguments.replace(/\{\{/g, "\\{\\{").replace(/\}\}/g, "\\}\\}") : "{}";
+        args = params.arguments
+          ? params.arguments
+            .replace(/\{\{/g, "\\{\\{")
+            .replace(/\}\}/g, "\\}\\}")
+            .replace(/\\\"/g, '"')
```

Comment:
This feels oddly specific, and like it might break things in different situations